admin/includes/init_includes/init_languages.php

```diff
@@ -58,7 +58,9 @@
 
 // include additional files:
 require (DIR_WS_LANGUAGES . $_SESSION ['language'] . '/' . FILENAME_EMAIL_EXTRAS);
-include (DIR_WS_LANGUAGES . $_SESSION ['language'] . '/widgets_default.php');
+if (file_exists(DIR_WS_LANGUAGES . $_SESSION ['language'] . '/widgets_default.php')) { 
+  include (DIR_WS_LANGUAGES . $_SESSION ['language'] . '/widgets_default.php');
+}
```

Comment:
What are your thoughts on this accommodating a fallback to default if the lang-specific version isn't found? ie: default to `''` if folder+file isn't found. That way the widgets wouldn't show up with no text in them.

Actually, as I look at the rest of this file, I guess strictly speaking that's a larger refactoring exercise for this entire file, and not really necessary for this specific PR.